src/globus_cli/commands/flows/start.py

```diff
@@ -78,6 +200,16 @@
         to create a list of tags.
     """,
 )
+@click.option(
+    "--activity-notification-policy",
+    type=ActivityNotificationPolicyType(),
+    help="""
+        The activity notification policy for the run.
+
+        This may be given as a JSON file or JSON data containing a full policy
+        document, or a comma-delimited list of statuses for notification.
```

Comment:
Current phrasing makes it sound like JSON is preferred, but I don't think that's what we should suggest.

How about:

```suggestion
        This may be given as a comma-delimited list of statuses for notification;
        alternatively, this can also be provided as JSON data--or a path to a
        JSON file--containing a full notification policy document.
```